portfolio/src/main/webapp/script.js

```diff
@@ -11,3 +11,19 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+$(document).ready(function(){
+    $('#2048-pic').hide();
+    $('#tweeter-pic').hide();
+    $('#job-pic').hide();
+    $('#mmr-pic').hide();
+    handleClickProject();
+});
+
+const handleClickProject = function(){
```

Comment:
generally put a space between () and { e.g.

const handleClickProject = function() {